src/components/posts/PostCard.jsx

```diff
@@ -0,0 +1,188 @@
+import { useState, useEffect } from 'react'
+import styled from 'styled-components'
+import { useUser } from 'libs/recoil/hooks'
+import { Neumorphism } from 'components/commons/Neumorphism'
+import { Glassmorphism } from 'components/commons/Glassmorphism'
```

Comment:
```suggestion
import { Neumorphism, Glassmorphism, Slider } from 'components/commons'
```
commonsのimport文はまとめて書ける気がします。